seatunnel-core/seatunnel-core-starter/src/main/java/org/apache/seatunnel/core/starter/execution/PluginUtil.java

```diff
@@ -98,10 +104,11 @@ public static SourceTableInfo createSource(
                     CatalogTableUtil.convertDataTypeToCatalogTables(seaTunnelDataType, tableId);
         }
 
-        if (catalogTables.size() != 1) {
-            throw new SeaTunnelException(
-                    String.format("Unsupported table number: %d on flink", catalogTables.size()));
-        }
+        //  if (catalogTables.size() != 1) {
+        //      throw new SeaTunnelException(
+        //              String.format("Unsupported table number: %d on flink",
+        // catalogTables.size()));
+        //  }
```

Comment:
why？

Reply:
This is specifically for multi-table exceptions and can be removed when both spark and flink are implemented

Reply:
flink is currently not supported

Reply:
It affects not only flink but also spark

Reply:
yes, I think flink is about to implement multiple tables as well

Reply:
Ok. Please clean the code.

Reply:
Let's clean it up when flink implements multiple tables